Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml

```diff
@@ -146,32 +161,58 @@ script:
     - contextPath: WildFire.Report.URL
       description: URL of the submission.
       type: string
-    description: Uploads the URL of a remote file to WildFire for analysis.
-  - name: wildfire-report
-    arguments:
-    - name: md5
+  - arguments:
+    - default: false
       description: MD5 hash to check.
       isArray: true
-    - name: sha256
+      name: md5
+      required: false
+      secret: false
+    - default: false
       description: SHA256 hash to check
       isArray: true
-    - name: hash
+      name: sha256
+      required: false
+      secret: false
+    - default: false
       description: Deprecated - Use the sha256 argument instead.
       isArray: true
-    - name: format
-      auto: PREDEFINED
+      name: hash
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: pdf
+      description: Request a structured report (XML or PDF).
+      isArray: false
+      name: format
       predefined:
       - xml
       - pdf
-      description: Request a structured report (XML PDF).
-      defaultValue: pdf
-    - name: verbose
-      auto: PREDEFINED
-      predefined:
-        - "true"
-        - "false"
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
       description: Receive extended information from WildFire.
-      defaultValue: "false"
+      isArray: false
+      name: verbose
+      predefined:
+      - 'true'
+      - 'false'
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        Retrieves results for a url using WildFire. Supported currently only in US cloud. Report format is in json.
```

Comment:
Retrieves results for a URL using WildFire. Currently, this is only supported in US cloud. The report format is in JSON.